dxw/govuk-blogs
# Chore/migrate to php8.3

This updates the theme for use in PHP 8. The only changes to production files are composer updates.

## How to test

Because this is a standalone theme, it's slightly more difficult to test. But one way is to check out the draft PR for updating govuk-blogs as a whole to PHP8: https://github.com/dxw/gds-blogs/pull/171, then `cd` into the theme directory and checkout this branch, and navigate around the site generally. I've done that locally, and aren't seeing any issues in the browser or in the logs.
